docs/querying/api.md

```diff
@@ -1393,3 +1393,54 @@ Enable the OTLP receiver by setting
 endpoint is `/api/v1/otlp/v1/metrics`.
 
 *New in v2.47*
+
+## Notifications
+
+The following endpoints provide information about active notifications in
+Prometheus.
+
+## Notifications
+
+The following endpoints provide information about active notifications in Prometheus.
+These endpoints are **experimental**. They may change in the future.
+
+### Active Notifications
+
+The `/notifications` endpoint returns a list of all currently active notifications.
+
+```bash
+GET /api/v1/notifications
+```
+
+Example:
+
+```bash
+$ curl http://localhost:9090/api/v1/notifications
+{"status":"success","data":[{"text":"Prometheus is shutting down and gracefully stopping all operations.","date":"2024-10-07T12:33:08.551376578+02:00","active":true}]}
```

Comment:
We JSON-pretty-print all the other API outputs for readability, let's do this here and below as well.